specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json

```diff
@@ -3016,7 +3016,7 @@
           }
         },
         "vrOpsEnabled": {
-          "description": "Is Vrops enabled/disabled",
+          "description": "Is vRealize Operations Manager enabled/disabled",
```

Comment:
How do you get this name `vRealize`?

Reply:
I googled 'vrops' and guessed it refers to this. Though apparently its the old product name, now. https://www.vmware.com/products/vrealize-operations.html.